product-mini/platforms/posix/main.c

```diff
@@ -387,6 +405,49 @@ main(int argc, char *argv[])
             }
             func_name = argv[0];
         }
+#if WASM_ENABLE_JIT != 0 || WASM_ENABLE_FAST_JIT != 0
+        else if (!strcmp(argv[0], "--interp")) {
+            init_args.running_mode = Mode_Interp;
```

Comment:
Define running_mnode = 0 at the begining of the function and here `running_mode = Mode_Interp`.